frontend/appflowy_flutter/lib/shared/icon_emoji_picker/icon_picker.dart

```diff
@@ -78,31 +79,57 @@ class FlowyIconPicker extends StatefulWidget {
     super.key,
     required this.onSelectedIcon,
     required this.enableBackgroundColorSelection,
+    this.iconPerLine = 9,
   });
 
   final bool enableBackgroundColorSelection;
   final ValueChanged<IconsData> onSelectedIcon;
+  final int iconPerLine;
 
   @override
   State<FlowyIconPicker> createState() => _FlowyIconPickerState();
 }
 
 class _FlowyIconPickerState extends State<FlowyIconPicker> {
-  late final Future<List<IconGroup>> iconGroups;
+  final List<IconGroup> iconGroups = [];
+  bool loaded = false;
   final ValueNotifier<String> keyword = ValueNotifier('');
   final debounce = Debounce(duration: const Duration(milliseconds: 150));
 
+  Future loadIcons() async {
```

Comment:
```suggestion
  Future<void> loadIcons() async {
```

Move this function below the build function